scio-avro/src/main/scala/com/spotify/scio/avro/AvroIO.scala

```diff
@@ -88,20 +88,25 @@ object ObjectFileIO {
   val WriteParam = AvroIO.WriteParam
 }
 
-final case class ProtobufIO[T: ClassTag: Coder](path: String)(implicit ev: T <:< Message)
+final case class ProtobufIO[T: ClassTag](path: String)(implicit ev: T <:< Message)
     extends ScioIO[T] {
   override type ReadP = Unit
   override type WriteP = ProtobufIO.WriteParam
   override final val tapT = TapOf[T]
 
+  private val protoCoder =
+    Coder
+      .protoMessageCoder[Message](classTag[T].asInstanceOf[ClassTag[Message]])
```

Comment:
The fact the we use an evidence of `T <:< Message` instead of a proper type constraint makes that awkward. I think we should rework that but it's out of the scope of this PR.